launch:
- node:
    pkg: "traffic_planner"
    exec: "planner"
    name: "example_traffice_planner"
    ros__parameters:
      map_location: "./traffic_planer_data/map.yaml"
      travel_plan_location: "/traffic_planer_data/travel_plan.yaml"
      switch_service_name: "switch_controller"
      tracker_topic_name: "tracked_objects"